Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1300,7 +1309,10 @@ - (void)closeAllMediaPresentations:(void (^)(void))completionHandler
 - (void)closeAllMediaPresentationsWithCompletionHandler:(void (^)(void))completionHandler
 {
     THROW_IF_SUSPENDED;
-    auto callbackAggregator = CallbackAggregator::create(WTFMove(completionHandler ?: ^{ }));
+    auto callbackAggregator = CallbackAggregator::create([completionHandler = makeBlockPtr(completionHandler)] {
```

Comment:
I think you can make less changes on this one?
```suggestion
    auto callbackAggregator = CallbackAggregator::create(WTFMove(completionHandler ?: [] { }));
```

Reply:
I don’t think so, since `completionHandler` would still be a block